scripts/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:latest
+
+WORKDIR /ws
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone && \
```

Comment:
Do you know why this TZ hacking is needed.. seems a little odd.

Reply:
no idea, ask #1971 ? this PR is based on that one.